implementations/rust/ockam/ockam_command/src/node/start.rs

```diff
@@ -12,8 +12,7 @@ use crate::{help, node::HELP_DETAIL, CommandGlobalOpts};
 #[command(arg_required_else_help = true, after_long_help = help::template(HELP_DETAIL))]
 pub struct StartCommand {
     /// Name of the node.
-    #[arg(default_value = "default")]
-    node_name: String,
+    node_name: Option<String>,
```

Comment:
This could be rewritten as: `#[arg(default_value_t = "default_node_name")]` so we don't have the `match cmd.node_name` in the command logic.

Reply:
You are right, this would be much cleaner. I believe you mean something like 
```
#[arg(default_value_t = default_node_name())]
node_name: String,
```
to get the default node from [`default_node_name`](https://github.com/Mo-Fatah/ockam/blob/01668d12ea1a7b6648754b7daf1bbe9f1d030b48/implementations/rust/ockam/ockam_command/src/node/util.rs#L243). But we will need to somehow pass the `opts` parameter when we call [`default_node_name`](https://github.com/Mo-Fatah/ockam/blob/01668d12ea1a7b6648754b7daf1bbe9f1d030b48/implementations/rust/ockam/ockam_command/src/node/util.rs#L243) to be something like 
```
#[arg(default_value_t = default_node_name(&opts))]
node_name: String,
```
Is there a workaround for this ?

Reply:
Yeah, `CliState` is a static object, so you can re-build it from scratch calling `CliState::new()`.